package.json

```diff
@@ -1,51 +1,51 @@
 {
-  "name": "talktalk-node",
-  "version": "1.0.0",
-  "description": "TalkTalk Server and Client built on top of Prisma GraphQL",
-  "scripts": {
-    "build": "prisma generate && babel -d . src --out-dir build && cp -r src/schemas build/schemas",
-    "build:window": "prisma generate && babel -d . src --out-dir build && ncp src/schemas build/schemas",
-    "start": "npm run build && node build/server.js",
-    "start:window": "npm run build:window && node build/server.js",
-    "prepublishOnly": "npm run build",
-    "lint": "eslint .",
-    "test": "jest",
-    "test:coverage": "jest --coverage",
-    "test:watch": "jest --watch",
-    "git:log": "git log --decorate --oneline --all --graph",
-    "check-style": "eslint --quiet src/**/*.js"
-  },
-  "keywords": [],
-  "author": "",
-  "license": "dooboolab",
-  "dependencies": {
-    "dotenv": "^6.2.0",
-    "express-jwt": "^5.3.1",
-    "graphql": "^14.0.2",
-    "graphql-yoga": "^1.16.7",
-    "jsonwebtoken": "^8.4.0",
-    "prisma-binding": "^2.2.11",
-    "sha256": "^0.2.0"
-  },
-  "devDependencies": {
-    "@babel/cli": "^7.2.3",
-    "@babel/core": "^7.2.2",
-    "@babel/plugin-proposal-class-properties": "^7.2.3",
-    "@babel/plugin-proposal-json-strings": "^7.2.0",
-    "@babel/preset-env": "^7.2.0",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-eslint": "^10.0.1",
-    "babel-jest": "23.6.0",
-    "eslint": "^5.10.0",
-    "eslint-config-standard": "^12.0.0",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-node": "^8.0.0",
-    "eslint-plugin-promise": "^4.0.1",
-    "eslint-plugin-standard": "^4.0.0",
-    "graphql-cli": "^3.0.3",
-    "graphql-request": "^1.8.2",
-    "jest": "^23.6.0",
-    "ncp": "^2.0.0",
-    "prisma-cli": "^1.0.9"
-  }
-}
+    "name": "talktalk-node",
```

Comment:
The changes of `package.json` come from previous `PR`. I think this should be deleted.